src/themes/themes.story.tsx

```diff
@@ -109,8 +110,8 @@ function HookedStory() {
 }
 
 function ThemeBlock({ block }) {
-  const { theme } = useContext(ThemeContext);
-  const section = Object.entries(theme[block]);
+  const { content } = useContext(ThemeContext);
+  const section = Object.entries(useContext(ThemeContext)[block]);
```

Comment:
Line 113 should get { block } in addition to content and then line 114 will just be Object.entries(block)

Reply:
Ah, because `block` is already a prop on line 111, it's a duplicate identifier on line 113. Unless I'm unaware of some syntax where you can deconstruct an object from the passed prop...